src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReferralHelper.java

```diff
@@ -34,6 +34,11 @@ public final class ReferralHelper {
 
     private static final String TRUE = "True";
     private static final String FALSE = "False";
+    private static final String NO_TRACK_OLD_NAME = "No Conciliation";
+    private static final String SHORT_TRACK_OLD_NAME = "Fast Track";

```

Comment:
I've seen the thread in slack but it might be worth adding a comment here to explain why these names have to be how they are. Without that context it looks very weird!

Reply:
Nice comment Jack.  Agreed ;)

Reply:
Agreed.